config/fixtures.js

```diff
@@ -0,0 +1,93 @@
+import { consensus } from "./contracts/consensus.js";
+import { usernames } from "./contracts/usernames.js";
+import { multiPayment } from "./contracts/multipayment.js";
+
+const fixtureConfig = {
+    "transfer": {
+        recipientAddress: "0x6F0182a0cc707b055322CcF6d4CB6a5Aff1aEb22",
+        value: "100000000",
+    },
+    "transfer-0": {
+        nonce: "123",
+        recipientAddress: "0xb693449adda7efc015d87944eae8b7c37eb1690a",
```

Comment:
for consistency I prefer to use 0x6F0182a0cc707b055322CcF6d4CB6a5Aff1aEb22 as much as possible as recipient, unless we want to differentiate certain transactions (e.g. a vote for a validator)